stylus/include/Header.styl

```diff
@@ -0,0 +1,30 @@
+.hDiv {
+    display: inline-block;
+    padding-top: 25px;
+    text-align: center;
+    width: 19.7%;
```

Comment:
Use flex-box styles here, don't hardcode widths.

Flexbox styling helps you get away from width, and inline-block.  Check out a tutorial online.

```
.Header
    display: flex
    > div
        flex: 1
```